securedrop/tests/test_crypto_util.py

```diff
@@ -19,6 +19,15 @@ def setUp(self):
     def tearDown(self):
         utils.env.teardown()
 
+    def test_word_lists_imports(self):
+        """Make sure word count for each list is correct and Guard
+        against regressions regarding not stripping file contents
+        correctly.
+        """
+        self.assertNotIn('',   crypto_util.words
```

Comment:
Actually you can just ignore the comment and fix the other things. When #1913 is merged in flake8 will be in CI and that will automate this convo ;)